package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "eslint-plugin-jane",
   "description": "Jane's ESLint plugin and configurations",
-  "version": "10.0.0",
+  "version": "11.0.0-dev.3",
```

Comment:
do we need `-dev.3`?

Reply:
No, once approved we will get a final version published. This is published as a dev build for testing.